build.py

```diff
@@ -425,18 +425,23 @@ def handle_build_node(self, n):
 
             # Strip out the functions from shared library builds.
             if '$SHLINK' in str(n.executor):
-                assert len(lines) == 3
+                # Linux or Windows
+                assert len(lines) == 3 or len(lines) == 5
 
                 # Run the check now. It doesn't need to happen at runtime.
                 assert lines[0] == 'SharedFlagChecker(target, source, env)'
                 SCons.Defaults.SharedFlagChecker(targets, sources, myEnv)
 
                 # We don't need this right now, so just assert that we don't. It can be added if we
                 # ever need it.
-                assert lines[2] == 'LibSymlinksActionFunction(target, source, env)'
+                assert lines[2] == 'LibSymlinksActionFunction(target, source, env)' or \
```

Comment:
In the future, try to follow this advice from pep8:

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

Also prefer putting the binary op at the beginning of the new line: https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator